### ISSUE TITLE
Add support react@19

### ISSUE THREAD

**Issue by rktyt** [fisshy/react-scroll#582]
Looks like it hasn't been added to peerDependencies yet.

**Comment by Peter-Easterbrook** [comment]
Got it working on Vercel with npm install --force

**Comment by moazzam-ali** [comment]
I'm also facing dependency discrepency using it with Nextjs 15 
please add suport for React 19

**Comment by yobottehg** [comment]
To "just get it working" without any guarantee that it really works and you do not want to use `--force` you can use the following workaround in the package.json:

```
"overrides": {
    "react-scroll": {
      "react": "$react",
      "react-dom": "$react-dom"
    }
  },
```

This will allow this dependency to use the direct react version you require.

**Comment by MikeyA-yo** [comment]
Please it's not hard they should just change the peer dependency version and i can't use it in my project now

**Comment by Peter-Easterbrook** [comment]
You could also try suffixing the installation with "--legacy-peer-deps". I.e.: npm i react-scroll --legacy-peer-deps

**Comment by fisshy** [comment]
Yes, sorry guys, I some how managed to lock myself out from npm, I've submited a support request to reset my 2fa.

**Comment by fisshy** [comment]
v1.9.2 is out now
